ui/components/MesheryPatterns.js

```diff
@@ -1513,15 +1513,19 @@ function MesheryPatterns({
     const { uploadType, name, url, file, designType } = data;
     let requestBody = null;
     switch (uploadType) {
-      case 'File Upload':
+      case 'File Upload': {
```

Comment:
Ideally, this is case insensitive.

Reply:
ok, I understand. I could use a 'toLowerCase()' integration, but the uploadType is already listed [here](https://github.com/meshery/meshery/blob/7b3def9f13ec53f46b67fe0f116f4dfafda45cef/ui/components/MesheryPatterns.js#L1519) as one of two. Do I need to modify that?